src/aiocomelit/api.py

```diff
@@ -160,11 +162,16 @@ async def _check_logged_in(self, host_type: str) -> bool:
 
         return logged
 
-    async def _login(self, payload: dict[str, Any], host_type: str) -> bool:
+    async def login(self) -> bool:
+        """Login to Serial Bridge device."""
+        payload = {"dom": self.device_pin}
+        return await self._login(payload)
```

Comment:
Here?

Reply:
For the serial bridge, but for VEDO ?

Reply:
This is for both

Reply:
> This is for both

Which is wrong, as the 2 devices have different payload.
Recheck current code ;-)

Reply:
Oh lol, I overlooked the dom/code 5-6 times

Reply:
I think we can leave library as it is in the end

Reply:
I think it will help if the login is at least at the shared part